pkgs/servers/http/nginx/modules.nix

```diff
@@ -518,17 +519,19 @@ let self = {
       mkdir -p "$(dirname "$NJS_SOURCE_DIR")"
       cp --recursive "${src}" "$NJS_SOURCE_DIR"
       chmod -R u+rwX,go+rX "$NJS_SOURCE_DIR"
-      export configureFlags="''${configureFlags/"${src}"/"$NJS_SOURCE_DIR/nginx"}"
+      export configureFlags="''${configureFlags/"${src}"/"$NJS_SOURCE_DIR/nginx"} --with-ld-opt='-lz'"
       unset NJS_SOURCE_DIR
     '';
 
-    inputs = [ which ];
+    inputs = [ which zlib ];
```

Comment:
The `quickjs` package seems to be required as well.

Reply:
AFAICT that is currently not enabled, the configure option is off by default

Reply:
nvm, that was a completely unrelated issue. I think this can be merged now.